solr/solr-ref-guide/src/circuit-breakers.adoc

```diff
@@ -94,16 +114,45 @@ Configuration for CPU utilization based circuit breaker:
 Note that this configuration will be overridden by the global circuit breaker flag -- if circuit breakers are disabled, this flag
 will not help you.
 
-The triggering threshold is defined in units of CPU utilization. The configuration to control this is as below:
+The triggering threshold is defined in percent CPU usage. A value of "0" maps to 0% usage 
+and a value of "100" maps to 100% usage.  This circuit breaker will trip when the CPU usage is
+equal to or greater than 75%:
 
 [source,xml]
 ----
 <str name="cpuThreshold">75</str>
 ----
 
+=== System Load Average Circuit Breaker
+This circuit breaker tracks system load average and triggers if the recent load average exceeds a configurable threshold.
+
+This is tracked with the JMX metric `OperatingSystemMXBean.getSystemLoadAverage()`. That measures the
+recent load average for the whole system. A "load average" is the number of processes using or waiting for a CPU,
+usually averaged over one minute. Some systems include processes waiting on IO in the load average. Check the
+documentation for your system and JVM to understand this metric. For more information, see the
+https://en.wikipedia.org/wiki/Load_(computing)[Wikipedia page for Load],
+
+Configuration for load average circuit breaker:
+
+[source,xml]
+----
+<str name="loadAverageEnabled">true</str>
+----
+
+Note that this configuration will be overridden by the global circuit breaker flag -- if circuit breakers are disabled, this flag
+will not help you.
+
+The triggering threshold is a floating point number matching load average. This circuit breaker will trip when the
```

Comment:
Not reviewing these changes -- will depend on Cassandra's review at some point

Reply:
Cassandra already reviewed the changes in March and I included her recommended changes. https://issues.apache.org/jira/browse/SOLR-15056?focusedCommentId=17307856&page=com.atlassian.jira.plugin.system.issuetabpanels%3Acomment-tabpanel#comment-17307856

Reply:
Cool, then this stands resolved